src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputWhitespace.java

```diff
@@ -85,6 +85,8 @@ private int nonVoid()
     private void testCasts()
     {
         Object o = (Object) new Object(); // ok
+        o = (java.lang.
+                Long) o; // ok
```

Comment:
Please add an error case that spans lines to show it still reports violation correctly.